packages/Python/lldbsuite/test/lang/swift/clangimporter/config_macros/Makefile.dylib

```diff
@@ -0,0 +1,8 @@
+LEVEL = ../../../../make
+SWIFT_OBJC_INTEROP := 1
+DYLIB_SWIFT_SOURCES = dylib.swift
+DYLIB_NAME = Dylib
+
+SWIFTFLAGS_EXTRA = -Xcc -DDEBUG=1 -Xcc -D -Xcc SPACE -Xcc -U -Xcc NDEBUG
```

Comment:
This should be SWIFTFLAGS_EXTRA*S*

@adrian-prantl I suppose this makes the test wrong?